packages/awslint/bin/awslint.ts

```diff
@@ -241,14 +242,17 @@ main().catch(e => {
   if (stackTrace) {
     console.error(e.stack);
   }
-  process.exit(1);
+  process.exitCode = 1;
 });
 
 async function loadModule(dir: string) {
   const ts = new reflect.TypeSystem();
   await ts.load(dir, { validate: false }); // Don't validate to save 66% of execution time (20s vs 1min).
   // We run 'awslint' during build time, assemblies are guaranteed to be ok.
 
+  // We won't load any more assemblies. Lock the typesystem to benefit from performance improvements.
+  ts.lock();
```

Comment:
This enables performance improvements in `jsii-reflect` that rely on a locked typesystem.